recipes/default.rb

```diff
@@ -32,11 +32,32 @@
   mode 00755
 end
 
-execute 'install homebrew' do
-  command homebrew_go
-  environment lazy { { 'HOME' => ::Dir.home(homebrew_owner), 'USER' => homebrew_owner } }
-  user homebrew_owner
-  not_if { ::File.exist? '/usr/local/bin/brew' }
+begin
+  template '/etc/sudoers.d/homebrew_sudo' do
```

Comment:
Fails if `/etc/sudoers.d` doesn't exist.

Something like...
```ruby
...

directory '/etc/sudoers.d' do
  action :nothing
end

template '/etc/sudoers.d/homebrew_sudo' do
  notifies :create, 'directory[/etc/sudoers.d]', :before
  ...
```
...might help/fix it